lib/orbit/import/import_certifications.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Orbit.Import.ImportCertifications do
```

Comment:
This file is missing tests.

Reply:
🤦 https://github.com/mbta/orbit/pull/203/files#diff-9d41583cd5e179dab2b68a3c23d935102e1b9c49e3a66831fb668f314c0c7ecdR1